### PR TITLE
Updated internals section of devguide by deleting the "Known Compiler-related Experiments"

### PR DESCRIPTION
**Pull request by Lincoln-developer**
Update internals section of devguide (issue #1159 )

The "Introducing New Bytecode"  and "Known Compiler-related Experiments" do not belong in the compiler page. This change updates the compiler page by removing the "Introducing New Bytecode" and "Known Compiler-related Experiments from the compiler page.

----
:books: Documentation preview :books:: https://cpython-devguide--1161.org.readthedocs.build/

### CODE REVIEW

**Comment by AA-Turner** [comment]
Mark noted that *Introducing New Bytecode* didn't belong on this page, rather to delete it entirely. Also note you've only removed the headings, not the body of the sections.

A

File: internals/compiler.rst [hunk 1]
```diff
@@ -650,11 +630,4 @@ References
 .. _The Zephyr Abstract Syntax Description Language.:
    https://www.cs.princeton.edu/research/techreps/TR-554-97
 
-.. [#skip-peephole] Skip Montanaro's Peephole Optimizer Paper
-   (https://legacy.python.org/workshops/1998-11/proceedings/papers/montanaro/montanaro.html)
-
-.. [#Bytecodehacks] Bytecodehacks Project
-   (https://bytecodehacks.sourceforge.net/bch-docs/bch/index.html)
```
Comment (willingc) [inline internals/compiler.rst:633]:
```suggestion
```

File: internals/compiler.rst [hunk 2]
```diff
@@ -650,11 +630,3 @@ References
 .. _The Zephyr Abstract Syntax Description Language.:
    https://www.cs.princeton.edu/research/techreps/TR-554-97
```
Comment (willingc) [inline internals/compiler.rst:632]:
```suggestion
```

**Comment by willingc** [comment]
@Lincoln-developer I've updated your PR to only contain the deletion of the experiments section. I am doing some larger changes and will handle the bytecode section.

Congrats @Lincoln-developer on your first contribution to the devguide. :tada:

**Pull request merged**